src/UIComponents/SaveModel.jsx

```diff
@@ -20,13 +20,28 @@ class SaveModel extends Component {
     columnDescriptions: PropTypes.array
   };
 
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      saveMessage: null
+    };
+  }
+
   handleChange = (event, field, isColumn) => {
     this.props.setTrainedModelDetail(field, event.target.value, isColumn);
   };
 
+  catchResponse = response => {
+    this.setState({ saveMessage: saveMessages[response.status] });
```

Comment:
Probably not crucial, but we could have a fallback message in case we get a status that's not recognised.